packages/nitro-protocol/hardhat-deploy/02-token.ts

```diff
@@ -2,8 +2,16 @@ import {HardhatRuntimeEnvironment} from 'hardhat/types';
 import {DeployFunction} from 'hardhat-deploy/types';
 
 const func: DeployFunction = async function(hre: HardhatRuntimeEnvironment) {
-  const {deployments, getNamedAccounts} = hre;
-  const {deploy} = deployments;
+  const {
+    deployments: {deploy},
+    getChainId,
+    getNamedAccounts,
+  } = hre;
+
+  if ((await getChainId()) === '4') {
```

Comment:
Why is Rinkeby isolated here? Shouldn't we blacklist any non-internal testnet?

Reply:
Probably, yeah